test/unit/bootstrap.js

```diff
@@ -71,15 +79,15 @@ global.document = {
             const img = new DOMElement();
             img.width = 10;
             img.height = 10;
+            Object.defineProperty(img, 'src', {
+                set: () => img.emitEvent('load'),
```

Comment:
why you add this emit?

Reply:
To test the loading of an image, using `THREE.TextureLoader` in `Fetcher`.

Reply:
To test sprite loading?
I try to know because I remove all `emitEvent` in last PR on unit test ?
it complicates to handle

Reply:
Good, so it's remove the ambiguity.

Reply:
Yes that's it ! Unfortunately I didn't see it in the last PR as `mock` was moved.